creating_packages/package_approaches.rst

```diff
@@ -164,62 +152,54 @@ This will translate to the cmake variables:
+    The above code assumes that the package will always use the default Visual Studio runtime (``MD``/``MDd``).
     To keep the package configurable for supporting static(MT)/dynamic(MD) linking with the VS runtime
-    library, do the following:
+    library, you can do the following:
 
-    - Keep the ``compiler.runtime`` setting, i.e. do not implement the ``configure()`` method removing it.
-    - Don't let the CMake helper define the ``CONAN_LINK_RUNTIME`` env-var to define the runtime, because
-      defining it by the consumer will cause it to be incorrectly applied to both the Debug and Release artifacts.
-      This can be done with a ``cmake.command_line.replace("CONAN_LINK_RUNTIME", "CONAN_LINK_RUNTIME_MULTI")``
-      to define a new variable.
+    - Keep the ``compiler.runtime`` setting, e.g. do not implement the ``configure()`` method removing it.
+    - Don't let the CMake helper define the ``CONAN_LINK_RUNTIME`` variable to define the runtime and define ``CONAN_LINK_RUNTIME_MULTI``
+      instead.
+    - In *CMakeLists.txt*, use the ``CONAN_LINK_RUNTIME_MULTI`` variable to correctly setup up the runtime for debug and release flags.
     - Write a separate ``package_id()`` methods for MD/MDd and for MT/MTd defining the packages to be built.
-    - In *CMakeLists.txt*, use the ``CONAN_LINK_RUNTIME_MULTI`` variable to correctly setup up the runtime for
-      debug and release flags.
 
-    All these steps are already coded in the repo https://github.com/memsharded/hello_multi_config and commented
-    out as **"Alternative 2"**
+    All these steps are already coded in the repo https://github.com/conan-io/examples/tree/master/features/multi_config and commented
+    out as **"Alternative 2"**.
 
-Also, you can use any custom configuration as they are not restricted. For example, if
-your package is a multi-library package, you could try doing something like:
+Also, you can use any custom configuration as they are not restricted. For example, if your package is a multi-library package, you could
+try doing something like:
 
 .. code-block:: python
 
     def package_info(self):
         self.cpp_info.regex.libs = ["myregexlib1", "myregexlib2"]
         self.cpp_info.filesystem.libs = ["myfilesystemlib"]
 
-These specific config variables will not be automatically applied, but you can directly use them
-in your consumer CMake build script.
+These specific config variables will not be automatically applied, but you can directly use them in your consumer CMake build script.
 
 .. note::
 
-    The automatic conversion of multi-config variables to generators is currently only implemented
-    in the ``cmake`` and ``txt`` generators. If you want to have support for them in another
-    build system, please open a GitHub issue.
+    The automatic conversion of multi-config variables to generators is currently only implemented in the ``cmake`` and ``txt`` generators.
```

Comment:
I think `visual_studio` & `visual_studio_multi` generators also support it for a while

Reply:
You are right. However, I am not sure if adding the ``*_multi`` generators here would be confusing, as those are used only for consumption. Maybe @memsharded can give us a hint here